### ISSUE TITLE
[a11y] Cannot edit first 3 digits of form width

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1566]
https://user-images.githubusercontent.com/37256050/168135074-6e8e185f-e17f-4689-a6bc-447e221817a6.mov

macOS, Safari

**Comment by maxxxxxdlp** [comment]
100 is the minimum value for that parameter, which is why it doesn't let you go below that

**Comment by maxxxxxdlp** [comment]
This is a generic bug that is applicable to everywhere in Specify 7 where we have an input field with a minimum value
The solution would be to enforce the "min" on-field focus loss rather than on typing, but there are some complexities with that too...

**Comment by grantfitzsimmons** [comment]
Could you add a (+100) and (-100) pixels button to the left and right of the field?

**Comment by grantfitzsimmons** [comment]
![msedge_zErGWePDPx](https://user-images.githubusercontent.com/37256050/168909606-498e4a8a-61f2-4d33-ad27-2e996864e4f7.gif)

**Comment by maxxxxxdlp** [comment]
?

**Comment by maxxxxxdlp** [comment]
> This is a generic bug that is applicable to everywhere in Specify 7 where we have an input field with a minimum value The solution would be to enforce the "min" on-field focus loss rather than on typing, but there are some complexities with that too...

CSIRO mentioned this issue during webinar today
